Ruby/Ruby.sublime-syntax

```diff
@@ -202,8 +202,10 @@ contexts:
       captures:
         1: punctuation.definition.constant.ruby
       push: try-regex
-    - match: '\b(nil|true|false)\b(?![?!])'
-      scope: constant.language.ruby
+    - match: '\b(nil)\b(?![?!])'
+      scope: constant.language.nil.ruby
```

Comment:
I'd argue to use `constant.language.null` with regards to https://github.com/sublimehq/Packages/issues/2711#issuecomment-774523108

Reply:
Fixed !